src/vivarium/framework/components/manager.py

```diff
@@ -70,19 +89,54 @@ def setup_components(self, builder, configuration):
 
 
 class ComponentInterface:
+    """The component manager interface made available from the builder."""
 
     def __init__(self, component_manager: ComponentManager):
         self._component_manager = component_manager
 
     def add_components(self, components: Sequence):
+        """Register new components with the system. Components are setup after
+        managers.
+
+        Parameters
+        ----------
+        components:
+          Instantiated components to register
```

Comment:
Ok I'm just going to be nitpicky here: do we want to end these parameter descriptions with a period or not? I vote period because that's what the numpydoc guide says but I don't feel super strongly as long as we're consistent

Reply:
Use periods!